src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -109,3 +110,36 @@ def run(self, args, manage_args):
             print(template.format(
                 list(stats['pooling'])[0], queue_name, workers, concurrency_, concurrency_ // workers
             ))
+
+
+class PillowResourceReport(CommandBase):
+    command = 'pillow-resource-report'
+    help = """
+    Report of pillow resources.
+    """
+
+    arguments = ()
+
+    def run(self, args, manage_args):
```

Comment:
Can we extract some small methods out of it? this method is pretty big.

Reply:
better?